guides/common/modules/con_acd_architecture.adoc

```diff
@@ -0,0 +1,117 @@
+This uses the Ansible galaxy as configured in the `ansible.cfg` file.
+====
+
+[id="{context}_application_definitions"]
+== Application Definitions
+
+Application definitions are blueprints of deployed applications.
+They are required next to an xref:{context}_ansible_playbooks[Ansible playbook] to create xref:{context}_application_instances[application instances].
+
+Application definitions are typically handled by administrative {ProjectName} users.
+They serve as a _blueprint_ providing easy self service for end users.
+
+Navigate to *Applications > App Definitions* to manage application definitions for application centric deployment.
+
+[id="{context}_application_instances"]
+== Application Instances
+
+An application instance is a preconfigured xref:{context}_application_definitions[application definition] and can be rolled out by end users.
+This automatically deploys and configures hosts based on the application definition and Ansible playbook.
+
+Based on one application definition, end users can configure and deploy multiple application instances.
+
+Navigate to *Applications > App Instances* to manage application instances for application centric deployment.
+
+[id="{context}_ansible_variables_and_parameter_hierarchy"]
+== Ansible Variables and Parameter Hierarchy
+
+There are different levels on how to set Ansible variables and parameters for a service/host.
+
+.Ansible variables
+* On the base, there are Ansible group variables for _all_ services as defined in the Ansible playbook.
+* Next, there are Ansible group variables defined for _all_ services in the application definition.
+Individual Ansible variables can be locked, removed, or added to the application definition.
+They overwrite the ones from the Ansible playbook.
+* An application definition also contains Ansible variables for _service groups_.
+These overwrite Ansible variables set for all services.
+* An application instance can overwrite any unlocked Ansible variables from the application definition.
+
+.Parameters
+* Managed hosts inherit parameters from multiple entities:
+** Global parameters
+** Organization parameters
+** Location parameters
+** Domain parameters
+** Subnet parameters
+** Operating system parameters
+** Host group parameters
+** Host parameters
+
++
+For example, domain level parameters overwrite both location and organization based parameters and host parameters overwrite any other defined parameter.
+
+[id="{context}_user_roles"]
+== User Roles
+
+The ACD plugin is used to deploy applications and requires sufficient {ProjectName} permissions.
+Depending on your environment, there might be two different groups of users in regard to _managing_ an application and _deploying_ an application:
+
+* An administrative group of users might be responsible for providing the tools to deploy an application, i.e. an Ansible playbook and application definition.
```

Comment:
It's more like a "that is" than "for example" in my opinion.
I would argue to leave it as is, as this section specifically addresses user permissions; so `ACD Manager` users have permissions to add, edit, and remove Ansible playbooks and application definitions.

Do you see "for example" because there are more tools necessary to deploy applications?

Reply:
Fixed.